src/player.h

```diff
@@ -223,15 +223,15 @@ class player : public Character
         int get_lift_assist() const;
 
         bool list_ammo( const item &base, std::vector<item::reload_option> &ammo_list,
-                        bool empty = true ) const;
+                        bool empty = true, bool fullReload = false ) const;
```

Comment:
For the future (no need to change here), `snake_case` is the preferred style in this codebase.

Reply:
got it